apps/storybook/docs/a_afnemen/3_starter-app.stories.mdx

```diff
@@ -0,0 +1,149 @@
+import {Meta} from '@storybook/blocks';
+
+<Meta title="Afnemen/Starter App"/>
+
+# Starter App
+
+## Inhoudstafel
+
+- [Doel](#doel)
+- [Bibliotheken](#bibliotheken)
+- [Aanpak](#aanpak)
+- [Opzet](#opzet)
+- [Scripts](#scripts)
+- [Tooling](#tooling)
+
+## Doel
+
+Onder [https://github.com/milieuinfo/uigov-starter-app](https://github.com/milieuinfo/uigov-starter-app) vind je een
+voorbeeld 'starter' project met een basis setup. Het is een WIP, gelieve via PR's wijzigingen of uitbreidingen aan
+te vragen.
```

Comment:
Mss interne link naar PR's (`/docs/bijdragen-git-branching-strategie--documentatie#pull-requests-prs`) en andere relevante links nuttig bij het bijdragen.

bv.: https://milieuinfo.github.io/uigov-builds/release/1.32.0/storybook/?path=/docs/bijdragen-git-branching-strategie--documentatie#pull-requests-prs

Reply:
Op termijn, momenteel wil ik de conventies voor monorepo-ontwikkeling en starter-app (externe toepassingen) niet persé gelijk zetten.
Starter-app zal ook in GitLab staan en via MR's gebeuren - wat de ontwikkelaars kennen.